modules/tinymce/src/core/test/ts/browser/dom/SerializerTest.ts

```diff
@@ -163,8 +163,8 @@ describe('browser.tinymce.core.dom.SerializerTest', () => {
     DOM.setHTML('test', '<input type="text" value="text" length="128" maxlength="129" />');
     assert.equal(ser.serialize(DOM.get('test')), '<input type="text" value="text" length="128" maxlength="129" />');
 
-    DOM.setHTML('test', '<form method="post"><input type="hidden" name="method" value="get" /></form>');
-    assert.equal(ser.serialize(DOM.get('test')), '<form method="post"><input type="hidden" name="method" value="get" /></form>');
+    DOM.setHTML('test', '<form method="post"><input type="hidden" name="formmethod" value="get" /></form>');
+    assert.equal(ser.serialize(DOM.get('test')), '<form method="post"><input type="hidden" name="formmethod" value="get" /></form>');
```

Comment:
I'd rename `ser` to `serializer` :)

Reply:
Again, not something related to this change.